examples/general/incorrect_matches_operation/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "incorrect_matches_operation"
+version = "2.4.3"
+authors = ["Dominik Czarnota <dominik.b.czarnota+dylint@gmail.com>"]
+description = "A lint to check for incorrect operators used with matches! macros"
+edition = "2021"
+publish = false
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[[example]]
```

Comment:
Because you're using `dylint_testing::ui_test` (not [`dylint_testing::ui_test_example`](https://docs.rs/dylint_testing/latest/dylint_testing/fn.ui_test_example.html)), this example should not be necessary.